capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -122,46 +134,85 @@ class ClubPage extends Component {
     const members = this.state.members.length && this.state.members.map(m => <UserCard key={m.id} user={m} club={this.state.club} removeMember={this.removeMember} />);
     const assignments = this.state.assignments.length && <div> {this.state.assignments.map(a => <AssignmentCard key={a.id} assignment={a} />)} </div>;
     return (
-      <div className='container text-center'>
-        {isOwner &&
-          <Link to={`/adminclubpage/${this.state.club.id}`}>
-            <Button className='admin-button' variant='secondary'>
-              Admin page
-            </Button>
-          </Link>
-        }
-        <div className='title'> {this.state.club.name} </div>
-        <div> Club Owner: </div>
-        <Row className='align-items-center justify-content-center'>
-          {owner}
-        </Row>
-        <div className='description'> {this.state.club.description} </div>
-        {bookTile}
-        {isOwner &&
-          <SearchBookModal
-            objectId={this.props.id}
-            update={this.handleBookChange}
-            text='Change the Club&quot;s Book'
-            putURL='/api/clubs'
-            type='club'
-            btnStyle='btn btn-primary mb-4 mt-4 mr-2' />
-        }
-        {assignments}
-        {isOwner &&
-          <Form onSubmit={this.handleAssignmentPost} id='assignment-post-form'>
-            <Form.Group controlId='formPostAssignment'>
-              <Form.Label> Post a new assignment! </Form.Label>
-              <Form.Control as='textarea' rows='3' placeholder='Enter assignment text...' />
-            </Form.Group>
-            <Button variant='primary' type='submit'> Submit </Button>
-          </Form>
+      <div >
```

Comment:
Even if this is due to autoformatting, I feel like the extra space here makes it inconsistent with the other elements below, like `<Row>`

Reply:
Ok